subsys/bluetooth/audio/bap_scan_delegator.c

```diff
@@ -225,33 +262,46 @@ static void net_buf_put_recv_state(const struct bass_recv_state_internal *recv_s
 }
 
 static void receive_state_updated(struct bt_conn *conn,
-				  const struct bass_recv_state_internal *internal_state)
+				  struct bass_recv_state_internal *internal_state)
 {
+#if CONFIG_BT_BAP_SCAN_DELEGATOR_LOG_LEVEL >= LOG_LEVEL_DBG
```

Comment:
Did you have issues with doing

```suggestion
if(CONFIG_BT_BAP_SCAN_DELEGATOR_LOG_LEVEL >= LOG_LEVEL_DBG)
```
?

Reply:
Yes some bsim build errors